ibmdbpy/learn/association_rules.py

```diff
@@ -72,22 +79,29 @@ def __init__(self, modelname = None, minsupport = None, maxlen = 5, maxheadlen =
         Attributes
         ----------
                
-        nametable: Gets set at fit step. The name of the optional table which contains a mapping between the items from the input table and the item names. 
+        nametable: Gets set at fit step. The name of the optional table which contains a mapping
+            between the items from the input table and the item names.
```

Comment:
```suggestion
        nametable
            Gets set at fit step. The name of the optional table which contains a mapping
            between the items from the input table and the item names.
```

If there's no linebreak between the attribute name and its description, the result doesn't look great:

![Bildschirmfoto 2020-03-21 um 11 42 09](https://user-images.githubusercontent.com/78236/77224819-93956100-6b69-11ea-8277-5c837122f6cf.png)